selfdrive/car/subaru/carstate.py

```diff
@@ -47,6 +50,11 @@ def update(self, cp, cp_cam, cp_body):
     ret.gearShifter = self.parse_gear_shifter(self.shifter_values.get(can_gear, None))
 
     ret.steeringAngleDeg = cp.vl["Steering_Torque"]["Steering_Angle"]
+
+    if self.car_fingerprint not in PREGLOBAL_CARS:
```

Comment:
no reason to gate this unless we had a better signal for PREGLOBAL

Reply:
preglobal doesn't have a counter